viaf.py

```diff
@@ -136,7 +136,7 @@ def weigh_contributor(cls, candidate, working_sort_name, known_titles=None, stri
         # the given author, make sure that one of the working names
         # shows up in a name record.
         if strict:
-            if len(match_confidences) == 0:
+            if not match_confidences:
```

Comment:
It's a knee-jerk to past assumptions of list-hood. I'll change it back. match_confidences is always a dict.